src/features/DisplayControls/NursingTasks/utils/NursingTasksUtils.js

```diff
@@ -198,3 +200,4 @@ export const isTimeWithinAdministeredWindow = (
 
   return enteredTimeInEpochSeconds <= timeWithinWindowInEpochSeconds;
 };
+export const getTimeInSeconds = (days) => days * 86400;
```

Comment:
we can rename this method to covertDaysToSeconds and move this to DateTimeUtils file as this is generic method

Reply:
changes have been made